README.md

```diff
@@ -680,10 +680,37 @@ stelace.roles.remove(roleId, [options], [callback])
 
 
 
+### Saved searches
+
+#### List saved searches
+
+stelace.savedSearch.list([queryParams], [options], [callback])
+
+#### Read a saved search
+
+stelace.savedSearch.read(savedSearchId, [queryParams], [options], [callback])
+
+#### Create a saved search
+
+stelace.savedSearch.create(data, [options], [callback])
```

Comment:
You may want to explain that `data` is the object expected with standard `POST /search` endpoint (`stelace.search.results` method) with `save` automatically set to `true`.

Users also need to know that additional `name` property is required, as we don’t want to force them to guess about the mapping to endpoints in API reference.

Reply:
Thanks, I'll add explanations for this function.